### ISSUE TITLE
[Bug]: `aria-selected` value in Select

### ISSUE THREAD

**Issue by kilobyte2007** [radix-vue/radix-vue#1114]
### Environment

```bash
v1.9.0
```

### Link to minimal reproduction

https://www.radix-vue.com/components/select.html

### Steps to reproduce

See the attributes inside the developer tools of the browser.

### Describe the bug

Right now, the SelectItem component has the `aria-selected` attribute set to `true` when the option is currently **highlighted**. If I understand this correctly from the specification, it should be true for the option currently **selected** (the value of the select).

This works correctly for the Combobox from what I see.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

### Context & Screenshots (if applicable)

<img width="919" alt="image" src="https://github.com/radix-vue/radix-vue/assets/1911258/4dfa415b-4526-42ec-968e-07e0860e42b4">

**Comment by zernonia** [comment]
Thanks for reporting this @kilobyte2007 ! This ARIA design was adapted from Radix UI. I'll look into it soon ya 😁
If you wanna help dig around this, feel free to do so! 🙏🏻

**Comment by kilobyte2007** [comment]
Hey @zernonia, just submitted a PR. An easy fix.
